docs/changehistory/NextVersion.md

```diff
@@ -44,4 +44,3 @@ alert(`${result.value} was ${result.inserted ? "inserted" : "already present"}`)
 A changeset represents the delta (i.e. the "set of changes") between two points on an iModel's timeline. It can be identified by two means: a [ChangesetId]($common) and a [ChangesetIndex]($common) - every changeset has both once it has been pushed to iModelHub. A `ChangesetId` is a string that is formed from the checksum of the contents of the changeset and its parent `ChangesetId`. A `ChangesetIndex` is a small sequential integer representing the position of the changeset on the iModel's timeline. Later changesets will always have a larger `ChangesetIndex` than earlier changesets. However, it is not possible to compare two `ChangesetId`s and tell anything about their relative position on the timeline.
```

Comment:
This looks wrong. These changes aren't in 2.18. Did you mean to empty out this file?

Reply:
There were two lines of whitespace at the end of the file and it looks like my autoformatter deleted one of them when I was resolving merge conflicts, otherwise the contents should be the same as in master. Do you want me to revert that change?

Reply:
No, I see now - thanks.